src/features/register-player/utils/title-form.tsx

```diff
@@ -7,7 +7,10 @@ interface FormTitleProps extends React.HTMLAttributes<HTMLHeadingElement> {
 export function FormTitle({ className, children, ...props }: FormTitleProps) {
   return (
     <h1
-      className={cn('mb-4 text-center text-[32px] text-neutral-950', className)}
+      className={cn(
+        'mx-auto mb-4 max-w-[46.023rem] text-center text-[32px] font-semibold leading-none text-neutral-950',
```

Comment:
Alterar `text-[32px]` para `text-3xl` (equivalente à 30px) , evitando perder a responsividade.

Caso for realmente necessário ser 32px, sugiro alterar para `text-[2rem]`.